src/ResourceManager/ServiceBus/Commands.ServiceBus/Utilities/ServiceBusClient.cs

```diff
@@ -527,117 +497,6 @@ public bool DeleteSubscription(string resourceGroupName, string namespaceName, s
         }
 
         #endregion Subscription
-
-        //#region EventHub
```

Comment:
@v-Ajnava why was all of this commented out? Nonetheless, it should be deleted instead of commented out